### PR TITLE
Added 1.5.3 version information

### PR DESCRIPTION
**Pull request by sraghunandan**

### CODE REVIEW

File: content/index.html [hunk 1]
```diff
@@ -357,6 +333,13 @@ <h4 class="title">Latest Release
                                         <a href="https://cwiki.apache.org/confluence/display/CARBONDATA/Apache+CarbonData+1.3.1+Release"
                                            class="whatsnew" target="_blank">what's new</a>
                                     </div>
+                                    <div class="block-row">
+                                        <a href="https://dist.apache.org/repos/dist/release/carbondata/1.3.0/"
```
Comment (sgururajshetty) [inline content/index.html]: All releases before 1.4.1 is nor working under the path https://dist.apache.org/repos/dist/release/carbondata/...

So such references can be removed

File: content/index.html [hunk 2]
```diff
@@ -488,7 +471,7 @@ <h4 class="title">Getting Started
                             to do is:</p>
                         <ol class="orderlist">
                             <li>
-                                <a href="https://dist.apache.org/repos/dist/release/carbondata/1.5.2/"
+                                <a href="https://dist.apache.org/repos/dist/release/carbondata/1.4.1/"
```
Comment (sgururajshetty) [inline content/index.html]: 1.5.3 path
https://dist.apache.org/repos/dist/release/carbondata/1.5.3/

File: content/index.html [hunk 3]
```diff
@@ -357,6 +333,13 @@ <h4 class="title">Latest Release
...
                                     </div>
+                                    <div class="block-row">
+                                        <a href="https://dist.apache.org/repos/dist/release/carbondata/1.3.0/"
+                                           target="_blank">Apache CarbonData 1.3.0</a>
+                                        <span class="release-date">Feb 2018</span>
+                                        <a href="https://cwiki.apache.org/confluence/display/CARBONDATA/Apache+CarbonData+1.3.0+Release"
+                                           class="whatsnew" target="_blank">what's new</a>
+                                    </div>
                             </div>
                             <div class="nextR">
```
Comment (sgururajshetty) [inline content/index.html:351]: Could not find any content related to CarbonData Release notes 0.2.0 in he index.html published. 

Can this content be removed/

**Comment by sgururajshetty** [comment]
LGTM

**Comment by sraghunandan** [comment]
merged to asf-site

**Pull request closed without merging**